packages/components/scripts/vendorism/custom/details.vendorism.js

```diff
@@ -29,39 +40,36 @@ const transformComponent = (path, originalContent) => {
     ['\'details--rtl', 'isRtl'],
   ], content);
 
-  // Add support for size attribute
-  content = addSectionAfter(
-    content,
-    '@property({ type: Boolean, reflect: true }) disabled = false;',
-    `
-  /** The details's size. */
-  @property({ reflect: true }) size: 'medium' | 'large' = 'medium';
-    `,
-  );
-
-  content = addSectionAfter(
-    content,
-    'details: true,',
-    `          'details--size-medium': this.size === 'medium',
-          'details--size-large': this.size === 'large',
-    `,
-  );
-
-  // Add support for contained attribute
-  content = addSectionAfter(
-    content,
-    '@property({ type: Boolean, reflect: true }) disabled = false;',
-    `
-  /** Draws the details as contained element. */
-  @property({ type: Boolean, reflect: true }) contained = false;
-    `,
-  );
+  content = addSectionsAfter([
+    // Add support for size attribute
+    [
+      '@property({ type: Boolean, reflect: true }) disabled = false;',
+      `/** The details's size. */
+  @property({ reflect: true }) size: 'medium' | 'large' = 'medium';`,
+      { newlinesBeforeInsertion: 2, tabsBeforeInsertion: 1 },
```

Comment:
The combination of newlines and tabs here means indention is messy all over the place in the final component. Mind to remove one of them to make indention align again?

Reply:
I changed adding a tab with adding two spaces for the addSectionAfter and the addSectionBefore methods